Dockerfile

```diff
@@ -41,6 +41,10 @@ RUN \
 COPY templates /etc/sensu/templates
 COPY bin /bin/
 
+RUN chgrp -R sensu /etc/sensu; mkdir -p /home/sensu; chown sensu:sensu /home/sensu
```

Comment:
Don't we need to create the group and user?

Reply:
It must be part of the package install. The sensu user is already created at the point this RUN is executed.